setup.py

```diff
@@ -52,10 +52,10 @@ def readme():
             "Cython==0.29.22",
             "edward2 @ git+https://github.com/google/edward2.git@a06c3abd8ec9aa4928aad6ae336e7c0324edcbc7",
             "matplotlib==3.4.1",
-            "numpy==1.19.2",
+            "numpy==1.19.5",
             "pandas==1.2.3",
             "seaborn==0.11.1",
-            "SQLAlchemy==1.4.6",
+            "SQLAlchemy==1.3.23",
             "tensorflow==2.4.0",
```

Comment:
Did you run a quick experiment to check that this change in these libraries does not crash anything in gryffin?

Reply:
Yes, ran the example in `examples/synthetic_functions/` and it seems to work fine